src/store/actions/new-create-appointment.ts

```diff
@@ -108,19 +108,19 @@ function generateBodyRequest(app: Editor): any {
 		: `${moment(app.start).format('LLLL')} - ${moment(app.end).format('LT')}`;
 
 	const virtualRoomMessage = app?.room?.label
-		? `-:::_::_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_::_:_::-\n${
+		? `${ROOM_DIVIDER}\n${
 				app.organizer.fullName ?? ''
 		  } have invited you to a virtual meeting on Carbonio Chats system!\n\nJoin the meeting now on ${
 				app.room.label
 		  }\n\n${
 				app.room.link
-		  } \n\nYou can join the meeting via Web or by using native applications:\n\nhttps://play.google.com/store/apps/details?id=com.zextras.team&hl=it&gl=US\n\nhttps://apps.apple.com/it/app/zextras-team/id1459844854\n\n-:::_::_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_:_::_:_::- \n`
+		  } \n\nYou can join the meeting via Web or by using native applications:\n\nhttps://play.google.com/store/apps/details?id=com.zextras.team&hl=it&gl=US\n\nhttps://apps.apple.com/it/app/zextras-team/id1459844854\n\n${ROOM_DIVIDER}\n`
```

Comment:
should this string be translated?

Reply:
Yes it should. We have a different task to review the texts inside the appointments. It will be managed there.
https://zextras.atlassian.net/browse/IRIS-4569